how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host programs and data in the form of smart contracts and perform computations on smart contracts in a secure and trustworthy way, while users have to trust that only at least two thirds of hosting nodes are honest.
+To achieve this, the technology behind the Internet Computer (IC) had to be reconceived from ground up, learning many lessons from earlier projects that failed to create a World Computer.
+
+Anyone can deploy a canister smart contract, or just canister, on the Internet Computer.
+A canister bundles Wasm program code and memory pages into a single unit.
+Canister smart contracts are executed in a replicated manner on many machines and can change the state of memory of the canister.
+
+At a high level, the IC's architecture builds on *subnets*, or *subnetworks*: The IC as a whole consists of many subnets, where each subnet is its own blockchain that operates concurrently with and independently of the other subnets.
+A subnet is a blockchain-based *replicated state machine*, that is, a virtual machine that holds state in a secure and non-tamperable manner.
+Each subnet hosts smart contracts, called *canister smart contracts*, or just *canisters*.
+New subnets can be created from nodes added to the IC to scale out the system, analogous to how traditional architectures such as public clouds scale out by adding machines.
+Such a scale-out architecture is unique in the blockchain space and allows for limitless scaling, i.e., combining the security and resiliency properties of blockchains with the scalability properties known from the public cloud.
```

Comment:
Uhhhh....playing devil's advocate here: aren't subnets kinda sorta somehow the same as shards, so not unique?